internal/builtin/providers/terraform/provider.go

```diff
@@ -39,7 +36,7 @@ func (p *Provider) ValidateProviderConfig(req providers.ValidateProviderConfigRe
 	return res
 }
 
-// ValidateDataResourceConfig is used to validate the data source configuration values.
+// ValidateDataStoreResourceConfig is used to validate the data source configuration values.
 func (p *Provider) ValidateDataResourceConfig(req providers.ValidateDataResourceConfigRequest) providers.ValidateDataResourceConfigResponse {
```

Comment:
The method name change in the comment looks wrong here.

Reply:
oops, got caught up in a rename op, thanks!